pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -396,6 +396,85 @@ func createSquashDictInnerAssertLenKeysHinter() (hinter.Hinter, error) {
 	return newSquashDictInnerAssertLenKeysHint(), nil
 }
 
+// SquashDictInnerCheckAccessIndex hint updates the access index
+// during the dictionary squashing process
+//
+// `newSquashDictInnerCheckAccessIndexHint` takes 1 operander as argument
+//   - `loopTemps` variable is a struct containing `index_delta_minus1` as
+//     a first field
+//
+// `newSquashDictInnerCheckAccessIndexHint` writes to the first field `index_delta_minus1`
+// of the `loop_temps` struct the result of `new_access_index - current_access_index - 1`
+// and assigns `new_access_index` to `current_access_index` in the scope
```

Comment:
could you add to this annotation an information about how does the loop_temps struct looks like?

Reply:
sure